src/main/scala/is/hail/utils/richUtils/RichArray.scala

```diff
@@ -3,4 +3,6 @@ package is.hail.utils.richUtils
 
 class RichArray[T](val a: Array[T]) extends AnyVal {
   def index: Map[T, Int] = a.zipWithIndex.toMap
+
+  def toFastIndexedSeq: IndexedSeq[T] = a
```

Comment:
I like this.

Reply:
Yeah, don't know why we didn't add something like this sooner.